tests/qunit/junitlogger.js

```diff
@@ -1,5 +1,5 @@
 (function() {
-    var count = 0, suiteCount = 0, currentSuite, currentTest, suites = [], assertCount, start, results = {failed:0, passed:0, total:0, time:0};
```

Comment:
Maybe revert this file. This isn't part of the PlayCanvas engine sourcebase but part of QUnit instead.